src/components/ResultContainer.tsx

```diff
@@ -138,6 +148,11 @@ function ResultContainer({ response }: { response: QueryResponse | null }) {
               res.dataset_portal_uri,
               res.num_matching_subjects,
               res.image_modals?.join(', '),
+              res.available_pipelines
+                ? Object.entries(res.available_pipelines).flatMap(([name, versions]) =>
+                    versions.map((version) => `${name} ${version}`).join(', ')
+                  )
+                : {},
             ].join('\t')
```

Comment:
seems this section could make use of a shared, generic function that also addresses the unwrapping above